### DISCUSSION TITLE
Add Polkadot as a Chain option in the Explorer

### DISCUSSION THREAD

**Discussion by gluneau** [WalletConnect/walletconnect-monorepo#2494] [Developer Support]
There are now wallets that support this chain, let's add it to the list of Chain filters for the Explorer

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2903505/7c479584-731f-4cb4-b71c-e3694f2dba21)

For reference: [Another day, Another chain](https://twitter.com/WalletConnect/status/1658147124360491008)

**Comment by finessevanes** [comment]
Sharing with our team. Thank you for this call out!
